easybuild/easyconfigs/l/Ludwig/ludwig-0.23.1_looser_test_tolerance.patch

```diff
@@ -0,0 +1,14 @@
+Reduce the test tolerance a little due to some failing regression tests
```

Comment:
Should we have a link to the mentioned upstream issue in the patch comments?
- https://github.com/ludwig-cf/ludwig/issues/367

Reply:
Sure, it didn't exist when I created the patch...updating